src/plugin.js

```diff
@@ -163,8 +166,16 @@ export default function plugin() {
 
     const style = createStyleNode(path, displayName, opts);
 
+    const { text, imports } = buildTaggedTemplate(
+      path,
+      nodeMap,
```

Comment:
so, if i'm reading this correctly, this means that we can't ever do e.g. circular references, which... i guess makes sense? but they'd work in "normal" CSS, right?

Reply:
yeah, It'd be easy enough to fix tho we could defer construction of the text til after everything is is parsed